docs/guides/cli.md

```diff
@@ -52,6 +53,63 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+#### Request a specific example from a resource
+
+You can request a specific example from the named of your document by using the `__example` query string paramenter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__example=exampleKey
+```
+
+#### Change the dynamic/static flag for a particular request
+
+You can override the `dynamic` keyword (which decides whether the generated example is static or dynamic) through the `__dynamic` query string parameter.
+
+```bash
+curl -v http://127.0.0.1:4010/pets/123?__dynamic=false
+```
+
+## Proxy
+
+This commands creates an HTTP server that will proxy all the requests to the specified upstream server. Prism will analyze the request coming in and the response coming back from the upstream server and report the discrepancies with what's declared in the provided OpenAPI document.
+
+```
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2
+
+[CLI] …  awaiting  Starting Prism…
+[HTTP SERVER] ℹ  info      Server listening at http://127.0.0.1:4010
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets
+[CLI] ●  note      POST       http://127.0.0.1:4010/pets
+[CLI] ●  note      GET        http://127.0.0.1:4010/pets/{petId}
+```
+
+By default, the output violations will be reported on the standard output, but you can also specify to embed these in the HTTP Response:
+
+```bash
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2 --log httpHeader
+… …
+
+curl -v -s http://localhost:4010/pets/10 > /dev/null
+
+< HTTP/1.1 404 Not Found
+< sl_validation: [{"severity":"Error","message":"The received media type does not match the one specified in the document"},{"location":["body"],"severity":"Error","code":"type","message":"should be object"}]
+```
+
+You can see there's a `sl_validation` header which is a JSON Payload with all the violations found in the response.
+
+```bash
+prism proxy https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml https://petstore.swagger.io/v2 --log httpResponse
```

Comment:
hm.. maybe `httpBody` or `httpPayload` insted of `httpResponse`? There is `httpHeader` above. And having `httpHeader` vs `httpResponse` makes it a bit confusing

Reply:
I'm resolving this comment since we're having the same discussion below.